Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -189,6 +189,7 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
     self.viewModel.outputs.metadataData
       .observeForUI()
       .observeValues { [weak self] data in
+        self?.metadataIconImageView.isHidden = data.iconImage == nil
```

Comment:
@theginbin this is why I wanted to have conversation about this. I actually disagree that It will be easier to test. I understand that change on any one of the data properties causes signal of entire data change. But because the data unit consist of *contextually dependent properties* I believe it will be simpler to describe in the unit test and easier to understand where actual problem is in case of bug happened.